MantidPlot/src/ApplicationWindow.cpp

```diff
@@ -6071,7 +6071,12 @@ void ApplicationWindow::prepareSaveProject() {
       projectname, *serialiser, windows, this);
   connect(m_projectSaveView, SIGNAL(projectSaved()), this,
           SLOT(postSaveProject()));
-  m_projectSaveView->show();
+  return m_projectSaveView->exec();
+}
+
+void ApplicationWindow::prepareSaveProject() {
+  auto result = execSaveProjectDialog();
```

Comment:
Do we need to capture the variable here? Can't we just not assign the function call to anything?